content-src/components/Base/Base.jsx

```diff
@@ -128,6 +129,7 @@ export class BaseContent extends React.PureComponent {
     return (
       <div>
         <div className={outerClassName}>
+          {this.props.isFirstrun && <StartupOverlay />}
```

Comment:
Was this change made for performance reasons?

Reply:
Hmm so I was originally moving it around to see if it would get it to render faster but in the end I don't think it makes a difference so I'll change it back